etc/examples/c10d-Parallelism-example/README.md

```diff
@@ -0,0 +1,96 @@
+# PyTorch Fashion MNIST distributed training example using native data parallelism
+
+This is a 2 convolutional layers model using native pytorch method **dist.init_process_group** and **DistributedDataParallel** to sync and join all the worker nodes.
+- Tested with Syncing using **Shared File System** where worker processes will share their location/group name information on a shared file. (currently using it on shared pvc among the ffdl learners)
+- Tested with GPU with both **Gloo** and **NCCL** Backend.
+- This example is able to handle multiple GPUs in one rank.
```

Comment:
I think we should mention also that this example will assign single rank per node only. This is mentioned into the top level list of examples but will be nice to have it here too.

Reply:
@sdmonov Makes sense.